types/jsspec__jsspec/tslint.json

```diff
@@ -0,0 +1,11 @@
+{
+    "extends": "dtslint/dt.json",
+    "rules": {
```

Comment:
Hi @HookyQR , thanks for your contribution. We prefer not to have rules turned off for new definitions unless absolutely necessary.

Reply:
On it. 👍

Reply:
Back to you.